### PR TITLE
Add a style prop #4

### PR DESCRIPTION
**Pull request by medonagy45**
fixing the Style issue

### CODE REVIEW

**Comment by medonagy45** [comment]
Add a class prop was added too

**Comment by xnt** [comment]
Sorry for the delay, @medonagy45 ! I apparently accidentally turned off notifications. I'll check this out early tomorrow. Thank you very much for your time and effort!!

File: src/index.js [hunk 1]
```diff
@@ -11,9 +11,9 @@ export default class SomethingNice extends PureComponent {
 
   render() {
     return (
-      <span>
-        { this.message }
+      <span style={this.props.style} className={this.props.className}>
```
Comment (xnt) [inline src/index.js:19]: Mind adding these to the prop types, please?

**Review by xnt — CHANGES_REQUESTED** [review]

**Comment by xnt** [comment]
Resolves #4

**Comment by xnt** [comment]
Resolves #5

File: src/index.js (hunk 1 above)
Comment (medonagy45) [reply src/index.js:19]: I hope i made it right

Comment (xnt) [reply src/index.js:19]: Hi @medonagy45 . Please look at https://github.com/xnt/react-something-nice/blob/0aa122402bb507a00ffc0ac50f458cb72c0e97df/src/index.js#L29 .

`PropTypes` is a validator that can help us check if a property is passed, its type, its shape, and sometimes even helps a bit with our IDE's autocompletion and some linting tools (i.e. checking for typos).

Here is some doc on PropTypes - https://reactjs.org/docs/typechecking-with-proptypes.html

Let me know if you think you can implement the checks for `style` and `className`. What you've done so far is great! So I have no problem with merging your change and then implementing that validation. That being said, I wouldn't like to steal the spotlight from you unless you're cool with it.

Thank you *so much* for your contributions!

Comment (medonagy45) [reply src/index.js:19]: Hi @xnt ,
I have made the changes , check it  and feed me back

**Review by xnt — APPROVED** [review]
Thanks 👍 !

**Pull request merged**